erts/doc/src/erlang.xml

```diff
@@ -533,6 +581,10 @@ Z = erlang:adler32_combine(X,Y,iolist_size(Data2)).</code>
         <pre>
 > <input>binary_to_float(&lt;&lt;"2.2017764e+0">>).</input>
 2.2017764</pre>
+        <p>The float string format is the same as the format for
+        <seeguide marker="system/reference_manual:data_types">Erlang
+        float literals</seeguide> except for that underscores are not
+        permitted.</p>
```

Comment:
Remove "for" => "... except that underscores are not permitted.".

(Disclaimer: I'm not a linguist.)